lib/propolis/src/hw/nvme/mod.rs

```diff
@@ -496,7 +496,11 @@ pub struct PciNvme {
 
 impl PciNvme {
     /// Create a new pci-nvme device with the given values
-    pub fn create(serial_number: String, log: slog::Logger) -> Arc<Self> {
+    pub fn create(
+        serial_number: String,
+        mdts: Option<u8>,
```

Comment:
Since we're hardcoding this arg in all call sites right now can we just move it into the body for now? As-is I think it can be a bit hard to get correct and ensure a valid value is given. Case in point, `MDTS` of 20 means we're saying a max size of `2^20 * CAP.MPSMIN (4K) = 4GiB` and not the intended `1MiB`.